includes/wc-admin-update-functions.php

```diff
@@ -67,7 +67,7 @@ function wc_admin_update_0230_db_version() {
  * Remove the note unsnoozing scheduled action.
  */
 function wc_admin_update_0251_remove_unsnooze_action() {
-	as_unschedule_action( WC_Admin_Notes::UNSNOOZE_HOOK, null, WC_Admin_Notes::QUEUE_GROUP );
+	as_unschedule_action( WC_Admin_Notes::UNSNOOZE_HOOK, null, 'wc-admin-data' );
```

Comment:
Optional, but we're using `wc-admin-data` in a few places now, so it would be nice to have a constant somewhere for this.

Reply:
I only see it in https://github.com/woocommerce/woocommerce-admin/blob/54b037501de15c732b5134cc11090fade35746f8/src/Schedulers/SchedulerTraits.php#L21.

I think it's OK to use the string literal here as we're just cleaning up - this value shouldn't/won't/can't change.